db/queries.js

```diff
@@ -198,6 +198,41 @@ module.exports.getItemName = async function (params, callback) {
     return res.rows[0].name;
 }
 
+module.exports.getUsernameFromID = async function (params, callback) {
+    try{
+        const statement = `
+		select 	user_name
+		from 	miscellaneous.usernames
+		where 	user_id = $1;`;
+        const args = [params.user_id];
+        const res = await db.query(statement, args, callback);
+        return res.rows[0].user_name;
+    } catch (err) {
+        console.log(err);
+    }
+}
+
+module.exports.updateUsername = async function (params, callback) {
+    try{
+        const statement = `
+		insert into 
+		miscellaneous.usernames (user_name, user_id)
+		values ($1, $2)
+        on conflict (user_id) do update
+		set user_name = $3;`;
+        let args;
+        //checks if username is empty
+        if (params.user_name === undefined) {
+           args = ["", params.user_id, ""];
```

Comment:
3 spaces for indentation ‽

Reply:
I'd suggest to `params.user_name = ""` instead. This makes it act like a guard clause. Then the line inside the else block will be moved outside (and the else block will be empty, hence remove it)

Reply:
done, thanks